dev/2018/spring/knights-to-kings/knights-to-kings.scss

```diff
@@ -0,0 +1,216 @@
+@import '../../../../versions/v4/static/scss/variables';
+@import '../../../../versions/v4/static/scss/mixins';
+@import url('https://fonts.googleapis.com/css?family=IM+Fell+Great+Primer');
+@import url('https://fonts.googleapis.com/css?family=Oswald:400');
+
+$serif: 'IM Fell Great Primer', 'Georgia', serif;
+$sans-serif: "Gotham SSm 7r", "Gotham SSm A", "Gotham SSm B", sans-serif;
+$sans-serif-cond: 'Oswald', 'Arial Narrow', sans-serif;
+$story-gold: #dcaa27;
+$story-white: #f1f1f1;
+$story-black: #252525;
+
+article.story {
+  .story-header {
+    margin: 0;
+  }
+
+  .story-deck {
+    background-color: $story-gold;
+    font-family: $serif;
+
+    h2 {
+      font-size: 28px;
+      margin-bottom: 20px;
+
+      @media(min-width: $screen-sm-min) {
+        font-size: 40px;
+      }
+    }
+  }
+
+  .heading {
+    color: $story-gold;
+    display: inline-block;
+    font-family: $sans-serif-cond;
+    font-size: 28px;
+    line-height: 1;
+    margin-bottom: 14px;
+  }
+
+  p {
+    font-size: 16px;
+    line-height: 1.4;
+
+    &.story-intro {
+      @media(min-width: $screen-sm-min) {
+        font-size: 18px;
+      }
+    }
+
+    &.byline {
+      font-style: italic;
+    }
+  }
+
+  .story-dropcap {
+    background-repeat: no-repeat;
+    background-size: 102px 143px;
+    margin-top: 10px;
+    padding-top: 10px;
+  }
+
+  .kings-court {
+    background-color: $story-black;
+    color: $story-white;
+    padding-bottom: 3rem;
+
+    .kings-court-header {
+      background-position: 50% 50%;
+      background-repeat: no-repeat;
+      color: $story-black;
+      display: block;
+      font-family: $serif;
+      font-size: 32px;
+      margin: -28px -20px;
+      padding: 10px;
+
+      @media(min-width: $screen-sm-min) {
+        left: calc(50% - 179px);
+        margin: 0;
+        padding: 30px;
+        position: absolute;
+        text-transform: uppercase;
+        top: -40px;
+        transform: rotate(-3deg);
+        width: 358px;
+      }
+    }
+
+    img {
+      @media(max-width: $screen-sm-min) {
+        max-width: 100px;
+      }
+    }
+
+    ul {
+      font-family: $sans-serif;
+      font-size: 12px;
+      font-weight: 400;
+      list-style: none;
+      margin-bottom: 2rem;
+      text-align: left;
+
+      @media(min-width: $screen-xs-min) {
+        font-size: 13px;
+        text-align: center;
+      }
+
+      li {
+        margin-bottom: 5px;
+      }
+    }
+  }
+
+  .splatter-left {
+    left: -290px;
```

Comment:
This negative x-positioning on these splatters will cause horizontal scrollbars on smaller screen sizes--add `overflow-x: hidden` to `article.story` as a quick fix.